authors.yml

```
@@ -0,0 +1,31 @@
+# For roles, choose from:
+#
+# - conceptualization
+# - formal analysis
+# - investigation
+# - methodology
+# - software
+# - supervision
+# - writing
+# - critical feedback
+# - data curation
+# - editing
+# - project administration
+# - resources
+# - validation
+# - visualization
+authors:
+  - name: First M. Last
+    orcid: 0000-0002-6473-0921
```

Should probably use blank orcids here
```suggestion
    orcid: 0000-0000-0000-0000
```